manifests/base/provisioners/helm/resources/deployment.yaml

```diff
@@ -62,22 +57,9 @@ spec:
             - "--storage-dir=/var/cache/bundles"
             - "--http-bind-address=127.0.0.1:8080"
             - "--http-external-address=https://$(HELM_PROVISIONER_SERVICE_NAME).$(HELM_PROVISIONER_SERVICE_NAMESPACE).svc"
-            - "--bundle-ca-file=/etc/pki/tls/ca.crt"
           ports:
             - containerPort: 8080
-          volumeMounts:
-            - name: bundle-cache
-              mountPath: /var/cache/bundles
```

Comment:
Is it not possible to keep the unrelated volumeMounts and volumes in the base?

Reply:
Yeah we can definitely leave them in. I'll move these back; I did this to maintain the list order after strategic merge but there are other ways to do that.